README.md

```diff
@@ -5,6 +5,7 @@ This repository contains Chocolatey Package Manager definitions for
 
 [![Latest version released](https://img.shields.io/chocolatey/v/docker.svg)](https://chocolatey.org/packages/docker)
 [![Package downloads count](https://img.shields.io/chocolatey/dt/docker.svg)](https://chocolatey.org/packages/docker)
+[![Build status](https://ci.appveyor.com/api/projects/status/yrojei8f42a8enpg/branch/master?svg=true)](https://ci.appveyor.com/project/ahmetalpbalkan/docker-chocolatey/branch/master)
```

Comment:
Oh, yes, this badge link must be updated once you have an account and connected this repo to AppVeyor. The svg sure is one of my builds.